api_v3/web/getFeed.php

```diff
@@ -73,6 +73,7 @@ function getRequestParameter($paramName)
 $entryId = getRequestParameter('entryId');
 $limit = getRequestParameter('limit');
 $ks = getRequestParameter('ks');
+$playerUrlType = getRequestParameter('playerUrlType');
```

Comment:
Additionally this should probably be the default from now on for every new feed.